crates/bevy_ui/src/ui_node.rs

```diff
@@ -1240,16 +1247,23 @@ impl From<RepeatedGridTrack> for Vec<RepeatedGridTrack> {
 ///
 /// The default `span` is 1. If neither `start` or `end` is set then the item will be placed automatically.
 ///
-/// Generally, at most two fields should be set. If all three fields are specified then `span` will be ignored. If `end` specifies an earlier
+/// Generally, at most, two fields should be set. If all three fields are specified then `span` will be ignored. If `end` specifies an earlier
```

Comment:
I dont believe that the added comma should be there, it flows better and makes more intuitive sense without it or slightly reworded.
```suggestion
/// Generally, two fields should be set at most. If all three fields are specified then `span` will be ignored. If `end` specifies an earlier
```